src/gui/vector/qgsattributesformproperties.cpp

```diff
@@ -1163,6 +1166,11 @@ void QgsAttributesDnDTree::onItemDoubleClicked( QTreeWidgetItem *item, int colum
       QPlainTextEdit *qmlCode = new QPlainTextEdit( itemData.qmlElementEditorConfiguration().qmlCode );
       qmlCode->setPlaceholderText( tr( "Insert QML code here…" ) );
 
+      // resize behaviour
+      bool resize = itemData.qmlElementEditorConfiguration().resizeFlag ;
+
+      QgsMessageLog::logMessage( QStringLiteral( "------jgr----:resize:onItemDoubleClicked: %1" ).arg( resize ) );
```

Comment:
Is this for production?

Reply:
No, not yet. Thanks @m-kuhn for looking into it. I was trying to enhance this widget, but I'm getting into trouble. Still learning how to work with it.